fvtr/gccgo/gccgo.exp

```diff
@@ -38,6 +38,15 @@ $at_dir/bin/$env(AT_TARGET)-gccgo is missing."
 	set rc 1
 }
 
+printit "Test if libgo.so supports split-stack..."
+set libs [ exec find ${at_dir} -type f -name libgo.so* ]
+foreach f ${libs} {
+	if {[catch {exec nm -s ${f} | grep morestack} err]} {
```

Comment:
@RajalakshmiS That grep is expecting many symbols with name `*morestack*`, i.e.:
```
000000000118b160 t __generic_morestack
000000000118b0a0 t __generic_morestack_set_initial_sp
000000000118cd44 t __morestack
000000000118bda0 t __morestack_allocate_stack_space
000000000118bc40 t __morestack_block_signals
00000000000000c0 B __morestack_current_segment
000000000118ad40 t __morestack_fail
0000000001a319b0 b __morestack_fullmask
000000000118cea4 t __morestack_get_guard
0000000000000008 B __morestack_initial_sp
000000000118c100 t __morestack_load_mmap
000000000118ceb4 t __morestack_make_guard
000000000118cd40 t __morestack_non_split
000000000118ae60 t __morestack_release_segments
00000000000000c8 B __morestack_segments
000000000118ceac t __morestack_set_guard
000000000118bcf0 t __morestack_unblock_signals
```

> What if there is some other symbol morestack?

If the symbol is also related to split-stack, that's fine.  Actually, I do expect the name of these symbols may change.
But if the extra symbol is not related to split-stack, that would cause a *partial* false success.  I emphasized "partial" because the following tests that compile and run the binaries could catch this error, e.g. issue #143, but they don't provide a good error message.

If we start to grep just for `__generic_morestack` and split-stack decides to change its implementation and stop using this symbol, this would give us a false failure.

Reply:
Thanks for the explanation. LGTM